### PR TITLE
Documentation: adds ability to build Read the Docs static site.

### PR DESCRIPTION
**Pull request by intelkevinputnam**
Signed-off-by: Kevin Putnam <kevin.putnam@intel.com>

### CODE REVIEW

File: cmd/extensions.rst [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Extensions
```
Comment (mythi) [inline cmd/extensions.rst:1]: we cannot have this file without modifying Makefile. The default build target expects everything under `cmd/` is a direcrory.

Do we need  `extensions.rst` here?

Comment (intelkevinputnam) [reply cmd/extensions.rst]: Moved to /docs for now.

**Comment by mythi** [comment]
@intelkevinputnam thanks! Can you please squash the commits again. I'll try running & merging this on Monday.

**Comment by intelkevinputnam** [comment]
@mythi Will do. FYI this PR does not have GitHub workflow for automated deployment.

**Review by mythi — APPROVED** [review]

**Comment by mythi** [comment]
tested a manual 'build' and checked the output. looks nice!

**Pull request merged**